### PR TITLE
fix: mention free plan cloud in welcome

### PR DESCRIPTION
**Pull request by vinckr**
@blueacidfrog
closes https://github.com/ory-corp/product/issues/316

### CODE REVIEW

File: docs/docs/index.mdx [hunk 1]
```diff
@@ -45,10 +45,11 @@ Ory differentiates from other vendors in the following key areas:
 
 Ory Cloud is the commercial offering of Ory and is built on top of our popular
 open source software. Our goal with Ory Cloud is to offer a planet-scale,
-low-latency, resilient, and secure service that is easy to use and set up. 
+low-latency, resilient, and secure service that is easy to use and set up.
 
 In short: Ory Cloud is the most convenient and straightforward way to run Ory.  
-Sign up at [console.ory.sh](https://console.ory.sh) for an account and create a free developer project.  
+Sign up at [console.ory.sh](https://console.ory.sh) for an account and create a
```
Comment (aeneasr) [inline docs/docs/index.mdx:51]:
```suggestion
Sign up at [console.ory.sh](https://console.ory.sh/registration) for an account and create a
```
